scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -2667,6 +2687,21 @@ static void refresh_cpus_flags(struct task_ctx *taskc,
 			break;
 		}
 	}
+	if (enable_cpuset) {
+		bpf_for(cpuset_id, 0, nr_cpusets) {
+			struct cpumask_wrapper* wrapper;
```

Comment:
Blank line.

Reply:
I think I figured out the whitespace convention (before if, before for, sometimes after for), but LMK if there's still places it's wrong.

Reply:
Hmmm... yeah, on machines with single LLC nodes, this is fine, but if there are multiple LLCs per node, the current `allow_node_aligned` condition can lead to unexpected behaviors as tasks can end up in a DSQ that a layer doesn't have CPUs on. But going back to this PR, if cpusets aren't aligned with LLCs, wouldn't we have a similar problem?

Reply:
> Hmmm... yeah, on machines with single LLC nodes, this is fine, but if there are multiple LLCs per node, the current allow_node_aligned condition can lead to unexpected behaviors as tasks can end up in a DSQ that a layer doesn't have CPUs on.

Will fix that in a separate PR, thx. 

> But going back to this PR, if cpusets aren't aligned with LLCs, wouldn't we have a similar problem?

Yeah, I was just able to produce that kind of behavior w/ my test setup, gonna have to add something to deal with that.